index.js

```diff
@@ -24,7 +22,7 @@ Toolkit.run(async (tools) => {
   tools.log.complete("NPM install run");
 
   const { code } = await require("@zeit/ncc")(
-    path.join(workspace, "index.js"),
+    path.join(tools.workspace, tools.inputs.entrypoint),
```

Comment:
GitHub Actions will fill in the default value of `entrypoint`, but if you prefer for explicit-ness, we can add:

```suggestion
    path.join(tools.workspace, tools.inputs.entrypoint || 'index.js'),
```